### DISCUSSION TITLE
Nie mogę wejść na konto z dostępem studenckim na azurre

### DISCUSSION THREAD

**Discussion by Gvmbit** [AzureCommunityPL/group#143] [Ogólna]
Cześć. jak w temacie. Czy da się ustalić, które konto jest podpięte do danego e-maila z domeną uczelnianą? To niestety nie koniec moich problemów, ponieważ żadne z nich nie pozwala mi na dostęp do oprogramowania na zajęcia, a jak próbuję dodać e-mail z domeną, to informuje mnie, że jest on już gdzieś przypisany. Mam podejrzenia, że jest to spowodowane faktem, że zapomniałem iż posiadam już jedno konto i zarejestrowałem drugie na tego samego e-maila. Nie chciało mnie przepuścić przez weryfikację, pojawiał się błąd. Przy trójetapowej weryfikacji (e-mail=>telefon=>e-mail) nie wysłało mi tego linku weryfikacyjnego. Zamiast tego pojawił się e-mail zawarty w screenshocie.
https://instrukcje.put.poznan.pl/tworzenie-konta-w-portalu-microsoft-azure/?fbclid=IwAR3GG6JusB88-ezKOMSW764sdpDMLXvlPIaL-AfqKrqNukI2C0okoKkUbKs mowa o 11 punkcie podanej instrukcji - nigdy taki e-mail się nie pojawił.
![microsoft](https://user-images.githubusercontent.com/74198025/98563561-9c2ed900-22ab-11eb-8f99-979607de6a7c.jpg)

**Comment by wi5nia** [comment]
Nie znam sie na tym uczelnianym systemie teraz wiec moja odpowiedz bedzie jak ja bym poszedl rozwiazac problem - moze warto sie skontaktowac z uczelnianym administratorem systemy zeby wyczyscil problemy z kontami?
